### PR TITLE
Ra 510 deploy only

### PR DESCRIPTION
**Pull request by ingeknudsen**

### CODE REVIEW

File: pkg/apis/radixvalidators/validate_ra.go [hunk 1]
```diff
@@ -133,6 +133,33 @@ func PublicImageComponentCannotHaveSourceOrDockerfileSet(componentName string) e
...
+// ComponentWithDynamicTagRequiresTagInEnvironmentConfigForEnvironment Error if image is set with dynamic tag and tag is missing
+func ComponentWithDynamicTagRequiresTagInEnvironmentConfigForEnvironment(componentName, environment string) error {
+	return fmt.Errorf(
+		"Component %s with %s on image requires an image tag set on environment config for environment %s",
+		componentName, radixv1.DynamicTagNameInEnvironmentConfig, environment)
+}
+
+// ComponentWithDynamicTagCannotHaveTagInEnvironmentConfigForUnmappedEnvironment Unmapped environment cannot have tag
+func ComponentWithDynamicTagCannotHaveTagInEnvironmentConfigForUnmappedEnvironment(componentName, environment string) error {
+	return fmt.Errorf(
```
Comment (keaaa) [inline pkg/apis/radixvalidators/validate_ra.go]: is this an requirement - why does a branch need to be mapped to env? (e.g. deploy only)

File: pkg/apis/radixvalidators/validate_ra.go [hunk 2]
```diff
@@ -279,6 +306,27 @@ func validateComponents(app *radixv1.RadixApplication) []error {
 			errs = append(errs, PublicImageComponentCannotHaveSourceOrDockerfileSet(component.Name))
 		}
 
+		if component.Image != "" &&
```
Comment (keaaa) [inline pkg/apis/radixvalidators/validate_ra.go]: move into own method(s) - easier to understand whats being validated

File: pkg/apis/radixvalidators/validate_ra.go (hunk 1 above)
Comment (ingeknudsen) [reply pkg/apis/radixvalidators/validate_ra.go]: Done

File: pkg/apis/radixvalidators/validate_ra.go (hunk 2 above)
Comment (ingeknudsen) [reply pkg/apis/radixvalidators/validate_ra.go]: Done

**Pull request merged**